src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsFeaturedVideoSound.java

```diff
@@ -50,21 +51,24 @@ public void adsFeaturedVideoWithoutSoundMercury() {
   }
 
   private void verifyFeaturedVideoWithSound(String pageUrl) {
-    AdsJWPlayerObject jwPlayerObject = new AdsJWPlayerObject(driver, pageUrl);
+    AdsBaseObject pageObject = new AdsBaseObject(driver, pageUrl);
+    AdsJWPlayerObject jwPlayerObject = new AdsJWPlayerObject(driver);
 
     jwPlayerObject.waitForAdPlaying();
-    jwPlayerObject.scrollToPlayer();
+    pageObject.scrollTo(driver.findElement(AdsJWPlayerObject.PLAYER_SELECTOR));
     jwPlayerObject.clickVolumeButton();
     jwPlayerObject.allowToPlayVideoForSomeTime(Duration.ofSeconds(3));
     Assertion.assertTrue(jwPlayerObject.wasSoundHeard());
   }
 
   private void verifyFeaturedVideoWithoutSound(String pageUrl) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "verifyFeaturedVideoWithoutSound" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)